### ISSUE TITLE
Update to TF 2.0

### ISSUE THREAD

**Issue by Guyanqi** [tf-encrypted/tf-seal#45]
Recently I'm trying to transfer my code from TF 1 to  2.0 version, apparently they changed a lot in 2.0 version, e.g. no session() any more. Should we convert the code to TF 2.0?

**Comment by justin1121** [comment]
Yeah we should! Most of the work should be in the examples but it shouldn't really be that much work.

**Comment by mortendahl** [comment]
Might be worth taking a peek at [this guy](https://github.com/tf-encrypted/tf-seal/issues/43) at the same time; I had some issues with it not recognising the name ever, but looks like that has been fixed with TF2.
